From d85e1e5fbd24545192d01614755e1bcf986a236b Mon Sep 17 00:00:00 2001
From: Guy Harris <guy@alum.mit.edu>
Date: Wed, 5 Nov 2014 11:18:51 -0800
Subject: [PATCH 3/6] More descriptive name for bpf_filter1().

Call it bpf_filter_with_aux_data(), to better indicate what it does.

Also expand some comments and clean up white space a bit.
---
 pcap-linux.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pcap-linux.c b/pcap-linux.c
index 6563e56..05edd1c 100644
--- a/pcap-linux.c
+++ b/pcap-linux.c
@@ -1690,7 +1690,7 @@ pcap_read_packet(pcap_t *handle, pcap_handler callback, u_char *userdata)
 
 	/* Run the packet filter if not using kernel filter */
 	if (!handlep->filtering_in_kernel && handle->fcode.bf_insns) {
-		if (bpf_filter1(handle->fcode.bf_insns, bp,
+		if (bpf_filter_with_aux_data(handle->fcode.bf_insns, bp,
 		                packet_len, caplen, &aux_data) == 0)
 		{
 			/* rejected by filter */
@@ -4063,7 +4063,7 @@ pcap_read_linux_mmap(pcap_t *handle, int max_packets, pcap_handler callback,
                         aux_data.vlan_tag = h.h2->tp_vlan_tci;
                         aux_data.vlan_tag_present = (h.h2->tp_vlan_tci || (h.h2->tp_status & TP_STATUS_VLAN_VALID));
 
-                        if (bpf_filter1(handle->fcode.bf_insns, bp, tp_len, tp_snaplen, &aux_data) == 0)
+                        if (bpf_filter_with_aux_data(handle->fcode.bf_insns, bp, tp_len, tp_snaplen, &aux_data) == 0)
                                 goto skip;
 
                 }
-- 
1.8.3.1

